crates/fj-viewer/Cargo.toml

```diff
@@ -12,11 +12,13 @@ categories.workspace = true
 
 [dependencies]
 bytemuck = "1.12.1"
+crossbeam-channel = "0.5.6"
 egui = "0.19.0"
 egui-wgpu = "0.19.0"
 fj-interop.workspace = true
 fj-math.workspace = true
 raw-window-handle = "0.4.3"
+rfd = "0.10.0"
```

Comment:
It looks like by default, RFD uses GTK3 on Linux: https://github.com/PolyMeilex/rfd/blob/7a0ab7a0ec756b8cb8e30fba242ca86c372ee996/Cargo.toml#L14

This works fine on my machine, but I'm using Gnome, which is GTK-based. I don't think it will work everywhere. Based on the RFD documentation, it seems like [XDG Desktop Portal might be the better choice](https://docs.rs/rfd/0.10.0/rfd/index.html#xdg-desktop-portal-backend). It [supports fewer features](https://docs.rs/rfd/0.10.0/rfd/index.html#state), but I don't think we're using any of those anyway?

We should probably disable default features here, and enable the [`xdg-portal`](https://github.com/PolyMeilex/rfd/blob/7a0ab7a0ec756b8cb8e30fba242ca86c372ee996/Cargo.toml#L17) feature.